plugins/source/aws/resources/services/ssoadmin/permission_boundaries.go

```diff
@@ -0,0 +1,58 @@
+package ssoadmin
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/aws/aws-sdk-go-v2/service/ssoadmin"
+	"github.com/aws/aws-sdk-go-v2/service/ssoadmin/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+func permissionBoundaries() *schema.Table {
+	tableName := "aws_ssoadmin_permission_boundaries"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/singlesignon/latest/APIReference/API_GetInlinePolicyForPermissionSet.html`,
```

Comment:
Should this be https://docs.aws.amazon.com/singlesignon/latest/APIReference/API_GetPermissionsBoundaryForPermissionSet.html